frontend/packages/insights-plugin/src/plugin.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Plugin,
+  DashboardsOverviewHealthPrometheusSubsystem,
+  ModelFeatureFlag,
```

Comment:
Nit: unnecessary type import, since we don't need to add any feature flags.

Please remove it here and update the `ConsumedExtensions` type alias.